<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CarRentalMainWindow</class>
 <widget class="QMainWindow" name="CarRentalMainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1486</width>
    <height>619</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CarRentalMainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="allTabs">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>9</y>
      <width>1091</width>
      <height>593</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>2</number>
    </property>
    <widget class="QWidget" name="carTab">
     <attribute name="title">
      <string>Car</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string/>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_7">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>All Cars</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableView" name="assignCarTableView">
           <property name="styleSheet">
            <string notr="true">border: 0;
text-align: center;
background: lightgrey;</string>
           </property>
           <property name="textElideMode">
            <enum>Qt::ElideMiddle</enum>
           </property>
           <property name="gridStyle">
            <enum>Qt::DotLine</enum>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="cornerButtonEnabled">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderCascadingSectionResizes">
            <bool>true</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>150</number>
           </attribute>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderCascadingSectionResizes">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderStretchLastSection">
            <bool>false</bool>
           </attribute>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="addCarButton">
           <property name="styleSheet">
            <string notr="true">background: rgb(78, 154, 6)</string>
           </property>
           <property name="text">
            <string>Add Car</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="editCarButton">
           <property name="styleSheet">
            <string notr="true">background: rgb(252, 175, 62)</string>
           </property>
           <property name="text">
            <string>Edit Car</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="removeCarButton">
           <property name="styleSheet">
            <string notr="true">background: rgb(164, 0, 0)</string>
           </property>
           <property name="text">
            <string>Remove Car</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="customerTab">
     <attribute name="title">
      <string>Customer</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>All Customers</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="assignCustomerTableView">
        <property name="styleSheet">
         <string notr="true">border: 0;
text-align: center;
background: lightgrey;</string>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::DotLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="addCustomerButton">
        <property name="styleSheet">
         <string notr="true">background: rgb(78, 154, 6);
</string>
        </property>
        <property name="text">
         <string>Add Customer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="editCustomerButton">
        <property name="styleSheet">
         <string notr="true">background: rgb(196, 160, 0);</string>
        </property>
        <property name="text">
         <string>Edit Customer</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="removeCustomerButton">
        <property name="styleSheet">
         <string notr="true">background: rgb(164, 0, 0)</string>
        </property>
        <property name="text">
         <string>Remove Customer</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="assignTab">
     <attribute name="title">
      <string>Assign</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout_6">
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Assignment inputs</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="3" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>To</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLineEdit" name="assignCarIDInput"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Assign</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Customer ID</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Car ID</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLineEdit" name="assignCustomerIDInput"/>
         </item>
         <item row="6" column="0">
          <widget class="QPushButton" name="assignCarButton">
           <property name="styleSheet">
            <string notr="true">background: rgb(114, 159, 207);
font-weight: bold;
</string>
           </property>
           <property name="text">
            <string>Assign Car</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_3">
        <property name="styleSheet">
         <string notr="true">font-weight: bold;</string>
        </property>
        <property name="title">
         <string>Unassign Car</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="2" column="0">
          <widget class="QPushButton" name="unAssignCarButton">
           <property name="styleSheet">
            <string notr="true">background: rgb(252, 175, 62)</string>
           </property>
           <property name="text">
            <string>Unassign Car</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLineEdit" name="assignCarTabDeleteAssignmentCarIDInput"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font-weight: bold;</string>
           </property>
           <property name="text">
            <string>Assignment ID</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>All Assignments</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="assignCarToCustomerConfirmationMessage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QTableView" name="allAssignmentTableView">
        <property name="styleSheet">
         <string notr="true">border: 0;
text-align: center;
background: lightgrey;</string>
        </property>
        <property name="textElideMode">
         <enum>Qt::ElideMiddle</enum>
        </property>
        <property name="gridStyle">
         <enum>Qt::DotLine</enum>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
        <property name="cornerButtonEnabled">
         <bool>false</bool>
        </property>
        <attribute name="horizontalHeaderCascadingSectionResizes">
         <bool>true</bool>
        </attribute>
        <attribute name="horizontalHeaderMinimumSectionSize">
         <number>150</number>
        </attribute>
        <attribute name="horizontalHeaderDefaultSectionSize">
         <number>200</number>
        </attribute>
        <attribute name="horizontalHeaderStretchLastSection">
         <bool>true</bool>
        </attribute>
        <attribute name="verticalHeaderVisible">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderCascadingSectionResizes">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderMinimumSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderDefaultSectionSize">
         <number>20</number>
        </attribute>
        <attribute name="verticalHeaderHighlightSections">
         <bool>false</bool>
        </attribute>
        <attribute name="verticalHeaderStretchLastSection">
         <bool>false</bool>
        </attribute>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="assignCarTabConfirmationMessage">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
    <widget class="QWidget" name="searchTab">
     <attribute name="title">
      <string>Search</string>
     </attribute>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Search Car</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTableView" name="searchCarResult">
            <property name="styleSheet">
             <string notr="true">border: 0;
text-align: center;
background: lightgrey;</string>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>150</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="registrationNumberLabel">
            <property name="text">
             <string>Registration Number:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchCarRegistrationNumberInput">
            <property name="styleSheet">
             <string notr="true">text-align: center;
background: lightgrey;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchCarButton">
            <property name="styleSheet">
             <string notr="true">border-radius: 10px;
text-align: center;
padding: 10px;
background: rgb(78, 154, 6);</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="searchCarConfirmationMessage">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Search Customer</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTableView" name="searchCustomerResult">
            <property name="styleSheet">
             <string notr="true">border: 0;
text-align: center;
background: lightgrey;</string>
            </property>
            <attribute name="horizontalHeaderMinimumSectionSize">
             <number>200</number>
            </attribute>
            <attribute name="horizontalHeaderDefaultSectionSize">
             <number>200</number>
            </attribute>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="searchCustomerFirstNameInputLabel">
            <property name="text">
             <string>First Name:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="searchCustomerFirstNameInput">
            <property name="styleSheet">
             <string notr="true">text-align: center;
background: lightgrey;</string>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchCustomerButton">
            <property name="styleSheet">
             <string notr="true">border-radius: 10px;
text-align: center;
padding: 10px;
background: rgb(78, 154, 6);</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="searchCustomerConfirmationMessage">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
